woocommerce/woocommerce-ios
# Bulk Editing: Add selection state to products list

Part of: https://github.com/woocommerce/woocommerce-ios/issues/8517

## Description

This PR adds bulk editing state on Products List.
It replaces navbar content, disables pull-to-refresh, switches table view into multiple selection mode.

## Testing

1. Build and run the app in debug/alpha mode.
2. On the products list tap the "Bulk Edit" button in the navbar.
3. Confirm navbar title changes, all buttons are replaced with "Cancel".
4. Confirm pull-to-refresh is unavailable.
5. Select a few products from the list.
6. Confirm title changes, displaying amount of selected items.
7. Tap "Cancel" in navbar.
8. Confirm normal navbar title and items are back, pull-to-refresh is enabled.
 
## Video

https://user-images.githubusercontent.com/3132438/210849180-41e04008-4ee8-4e4e-bf89-47b2a9d1131f.mp4

---
